LICENSE

```diff
@@ -1,13 +1,201 @@
-Copyright 2023 Core Mesa Team and contributors
```

Comment:
The original text is still necessary, except that it should be renamed to `NOTICE`, while having the `LICENSE` text with the new content in this PR. See Apache Spark's [NOTICE](https://github.com/apache/spark/blob/master/NOTICE) and [LICENSE](https://github.com/apache/spark/blob/master/LICENSE).

Reply:
cc: @wang-boyu for Mesa-Geo, @adamamer20 for mesa-frames to be consistent with the outcome of this PR.

Reply:
In the official license that’s moved to line 189.

Do you think it should be at the top?

Reply:
[L189](https://github.com/apache/spark/blob/067f8f188eb22f9abe39eee0d70ad1ef73f4f644/LICENSE#L189) is in the appendix of the LICENSE text, and is just a guide on how to apply it.

Reply:
I am okay with NOTICE if LICENSE points to the NOTICE.

Reply:
According to https://www.apache.org/foundation/license-faq.html#Apply-My-Software, it should be the other way around.
> Include a copy of the Apache License, typically in a file called LICENSE, in your work, and consider also including a NOTICE file.